sklearn/kernel_approximation.py

```diff
@@ -39,7 +39,7 @@ class RBFSampler(TransformerMixin, BaseEstimator):
         Number of Monte Carlo samples per original feature.
         Equals the dimensionality of the computed feature space.
 
-    random_state : int, RandomState instance or None, optional (default=None)
+    random_state : int, RandomState instance, default=None
```

Comment:
since `None` has a very specific meaning in the case of `random_state`, we should  still mention it as a possible value. It does not correspond to "nothing" in this case:

```suggestion
    random_state : int, RandomState instance or None, default=None
```

Reply:
Changes done.